artemis-features/src/main/resources/artemis.xml

```diff
@@ -32,21 +32,23 @@ under the License.
            MAPPED: mmap files
            NIO: Plain Java Files
        -->
-      <journal-type>ASYNCIO</journal-type>
+      <journal-type>NIO</journal-type>
```

Comment:
ASYNCIO should be the default it’s more performant and Artemis will fall back to NIO if not present

Reply:
Well, I synced with a created default broker.xml its default is actually NIO and avoid "urgly" comments on every broker start if asyncio is not supported